pkg/Microsoft.Private.PackageBaseline/packageIndex.json

```diff
@@ -1232,8 +1232,8 @@
       ],
       "BaselineVersion": "4.5.0",
       "InboxOn": {
-        "net45": "4.0.0.0",
         "netcoreapp3.0": "4.0.0.0",
+        "net45": "4.0.0.0",
```

Comment:
We want this to be in the compat pack. You'll want to ask someone knowledgeable about how to hook that up. A follow up PR would be OK too